extensions/librdkafka/docker_tests/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
this file would have a better place in the later PR which will contain the docker tests

Reply:
These are manual tests that are ran for the processor. I would not want a PR with a processor that does not involve the testing done for it.